ui/page/dexclient/add_dex_modal.go

```diff
@@ -21,34 +18,59 @@ import (
 
 const addDexModalID = "add_dex_modal"
 
-const testDexHost = "dex-test.ssgen.io:7232"
-
 type addDexModal struct {
 	*load.Load
 	modal            *decredmaterial.Modal
-	addDexServer     decredmaterial.Button
+	addDexServerBtn  decredmaterial.Button
 	dexServerAddress decredmaterial.Editor
 	isSending        bool
-	cert             decredmaterial.Editor
-	cancel           decredmaterial.Button
+	cancelBtn        decredmaterial.Button
 	materialLoader   material.LoaderStyle
+	cert             decredmaterial.Editor
+
+	// defaultAppPass is set after login or initialize to continue process of adding a DEX
+	// without requesting password input on UI
+	defaultAppPass string
+	appPassword    decredmaterial.Editor
```

Comment:
I'm not familiar with a concept of "default app pass"; just app pass.
```suggestion
	// dexClientPassword will be used if set, without requesting password input on UI.
	dexClientPassword string
	dexClientPasswordEditor    decredmaterial.Editor
```